src/Features/CSharp/Portable/Completion/KeywordRecommenders/NintKeywordRecommender.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.CodeAnalysis.Completion.Providers;
+
+namespace Microsoft.CodeAnalysis.CSharp.Completion.KeywordRecommenders
+{
+    internal sealed class NintKeywordRecommender : AbstractNativeIntegerKeywordRecommender
+    {
+        protected override RecommendedKeyword Keyword => new RecommendedKeyword("nint");
```

Comment:
`nameof(nint)`?

Reply:
Yes, we can make that change when the feature is included in the build.

---
In reply to: [401372085](https://github.com/dotnet/roslyn/pull/42976#discussion_r401372085) [](ancestors = 401372085)